stylus/ui-base/fonts.styl

```diff
@@ -20,6 +20,10 @@ $font-title
 $font-labor
     font-family  Lato, sans-serif
 
+$font-smoothing
+    -moz-osx-font-smoothing grayscale
```

Comment:
Why writing a placeholder? why not writing it as default style for fonts directly on the body? I don't see any usecase where you could re-use that. :/
Just curious really. 😄